apps/nextjs-website/src/pages/[productSlug]/overview.tsx

```diff
@@ -34,6 +35,7 @@ export const getStaticProps: GetStaticProps<OverviewPageProps, Params> = ({
 const OverviewPage = ({ hero }: OverviewPageProps) => (
   <>
     <Hero title={hero?.title || 'missing title'} subtitle={hero?.subtitle} />
+    <ApiViewer />
```

Comment:
The API viewer should not be in the overview page, but is should have a dedicated page